Mathlib/Data/Sym/Card.lean

```diff
@@ -179,4 +179,22 @@ theorem card_subtype_not_diag [Fintype α] :
 protected theorem card {α} [Fintype α] : card (Sym2 α) = Nat.choose (card α + 1) 2 :=
   Finset.card_sym2 _
 
+variable {a b : α}
+
+theorem univ_singleton : @univ (Sym2 ({a} : Finset α)) _ = {s(⟨a, by simp⟩, ⟨a, by simp⟩)} := by
+  ext p
+  simp_rw [mem_univ, mem_singleton, true_iff]
+  refine p.inductionOn fun ⟨x, hx⟩ ⟨y, hy⟩ ↦ ?_
+  rw [mem_singleton] at hx hy
+  simp [hx, hy]
+
+theorem univ_pair :
+    @univ (Sym2 ({a, b} : Finset α)) _ = {s(⟨a, by simp⟩, ⟨a, by simp⟩),
+      s(⟨a, by simp⟩, ⟨b, by simp⟩), s(⟨b, by simp⟩, ⟨b, by simp⟩)} := by
+  ext p
+  simp_rw [mem_univ, mem_insert, mem_singleton, true_iff]
+  refine p.inductionOn fun ⟨x, hx⟩ ⟨y, hy⟩ ↦ ?_
+  rw [mem_insert, mem_singleton] at hx hy
+  rcases hx with hx | hx <;> rcases hy with hy | hy <;> simp [hx, hy]
```

Comment:
... and those become `Finset.sym2_singleton` (which already exists) and `Finset.sym2_pair`